pkg/operator/controller/ingress/controller_test.go

```diff
@@ -337,6 +353,18 @@ func TestSetDefaultPublishingStrategyHandlesUpdates(t *testing.T) {
 			expectedResult: false,
 			expectedIC:     makeIC(spec(elb()), status(nlb())),
 		},
+		{
+			name:           "loadbalancer ELB connection idle timeout changed from unset to 2m",
+			ic:             makeIC(spec(elbWithIdleTimeout(&metav1.Duration{Duration: 2 * time.Minute})), status(elb())),
```

Comment:
Why is the test case initially `status(elb())` and not `status(elbWithIdleTimeout(nil))`?

Reply:
I'll add a test case for `status(elbWithIdleTimeout(nil))`.